tests/framework/clients/k3d/k3d.go

```diff
@@ -1,190 +0,0 @@
-package k3d
-
-import (
-	"context"
-	"fmt"
-	"os/exec"
-
-	"github.com/pkg/errors"
-	apisV1 "github.com/rancher/rancher/pkg/apis/provisioning.cattle.io/v1"
-	"github.com/rancher/rancher/tests/framework/clients/rancher"
-	"github.com/rancher/rancher/tests/framework/extensions/clusters"
-	"github.com/rancher/rancher/tests/framework/extensions/defaults"
-	"github.com/rancher/rancher/tests/framework/pkg/config"
-	"github.com/rancher/rancher/tests/framework/pkg/session"
-	"github.com/rancher/rancher/tests/framework/pkg/wait"
-	"github.com/rancher/wrangler/pkg/randomtoken"
-	"github.com/sirupsen/logrus"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/watch"
-	"k8s.io/client-go/rest"
-	"k8s.io/client-go/tools/clientcmd"
-)
-
-var importTimeout = int64(60 * 20)
-
-// CreateK3DCluster creates a minimal k3d cluster and returns a rest config for connecting to the newly created cluster.
-// If a name is not given a random one will be generated.
-func CreateK3DCluster(ts *session.Session, name, hostname string, servers, agents int) (*rest.Config, error) {
```

Comment:
I don't think we should delete k3d functionality just because we won't use it in the integration setup. Let's keep this for maybe a future usage of some sort.